### PR TITLE
Update release leads for 1.12

### PR DESCRIPTION
**Pull request by skonto**
# Changes

- Adds Eventing leads similar to https://github.com/knative/release/pull/210

### CODE REVIEW

**Review by ReToCode — COMMENTED** [review]
You can add me to the Serving rooster as well.

Should the Github handle be used to compare with the peribolos file?

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Knative project releases quarterly, on Tuesday of the 4th week of January, A
 | Release | Release Date | Serving             | Eventing                        | PKG cut    | Unpin repos
 | ------- | ------------ | ------------------- | --------------------------------| ---------- | -----------
 | v1.11   | 2023-07-25   | skonto, dsimansk    | creydr, pierDipi, Vishal-Chdhry | 2023-07-18 | 2023-07-26
-| v1.12   | 2023-10-24   | skonto, ReToCode    | TBD                             | 2023-10-17 | 2023-10-25
+| v1.12   | 2023-10-24   | skonto, ReToCode    | Calum Murray, creydr            | 2023-10-17 | 2023-10-25
```
Comment (pierDipi) [inline README.md:21]:
```suggestion
| v1.12   | 2023-10-24   | skonto, ReToCode    | Cali0707, creydr            | 2023-10-17 | 2023-10-25
```

File: ROSTER.md [hunk 2]
```diff
@@ -25,11 +25,12 @@ This roster is seeded with all approvers from Eventing workgroups, plus addition
 - lionelvillard
 - matzew
 - houshengbo
+- Calum Murray
```
Comment (pierDipi) [inline ROSTER.md:28]:
```suggestion
- Cali0707
```

Comment (skonto) [reply ROSTER.md:28]: That would be my next question do we want the github handles. I copied what you had there.

**Review by creydr — COMMENTED** [review]
/lgtm

**Pull request merged**